numerics/polynomial.hpp

```diff
@@ -201,6 +207,30 @@ class PolynomialInMonomialBasis : public Polynomial<Value_, Argument_> {
   friend operator*(
       PolynomialInMonomialBasis<L, A, l, E> const& left,
       PolynomialInMonomialBasis<R, A, r, E> const& right);
+#if PRINCIPIA_COMPILER_MSVC_HANDLES_POLYNOMIAL_OPERATORS
```

Comment:
Add a blank line and a comment
```C++
// Operators polynomial ± constant.
```
So that the semantic of the `V` or `A` is clear.

Reply:
Done, but not here, in the *real* declarations.  These are the friend declarations, and we keep them as compact/cryptic a possible.